modules/cnf-reducing-netqueues-using-pao.adoc

```diff
@@ -0,0 +1,349 @@
+
+In real-time or low latency systems all the unnecessary IRQs that are pinned to the isolated CPUs must be moved to reserved/housekeeping CPUs. For DPDK based workloads for example to achieve low latency the number of NIC queues should be reduced to just the number of reserved/housekeeping CPUs.
+
+In deployments with applications that require system/OCP networking or in mixed deployments with DPDK workloads, multiple queues are needed to achieve good throughput and the number of NIC queues should either be adapted or remain unchanged.
+Enabling this feature in the performance profile ensures all IRQs are moved from isolated to housekeeping CPUs preventing the housekeeping CPUs from running out of vectors/capacity.
+
+
+[id="adjust-netqueues-using-pao_{context}"]
+== Adjusting the net queues with the performance profile
+
+The performance profile lets you adjust the queue count for each network device.
+
+Supported net devices are:
+
+* Non virtual network devices
+
+* Network devices that support multiple queues (channels)
+
+Unsupported net devices are:
+
+* Pure software network interfaces
+
+* Block devices
+
+* Intel DPDK virtual functions
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Install the OpenShift CLI (`oc`).
+
+
+.Procedure
+
+. Log in to the OpenShift Container Platform cluster running the Performance Addon Operator as a user with `cluster-admin` privileges.
+
+. Create a performance profile that is appropriate for your hardware and topology. For more information see xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#cnf-tuning-nodes-for-low-latency-via-performanceprofile_cnf-master[Tuning nodes for low latency with the performance profile]. Alternatively, edit an existing performance profile using the following command:
++
+[source,terminal]
+----
+$ oc edit performanceprofile <your-profile-name>
+----
+
+. Populate the `spec` field profile with the `net` object. The object list may contain two fields:
+
+* `userLevelNetworking` (required) specifies a boolean flag which if true sets the queue count to the reserved CPU count for all supported devices. The default is false.
+* `Devices` (optional) specifies a list of devices that will have the queues set to the reserved CPU count. If the device list is empty the configuration applies to all network devices. The configuration is as follows:
+** `interfaceName`: Name supports shell-style wildcards which can be positive or negative.
+*** Example wildcard syntax is as follows: <string> .*
+*** Negative rules are prefixed with an exclamation mark. To apply the net queue changes to all devices other than the excluded list use  `!<device>` for example !eno1.
+* `vendorId`: Network device vendor ID represented as a 16 bit hexmadecimal number with a 0x prefix.
+* `deviceId`: Network device ID (model) represented as a 16 bit hexmadecimal number with a 0x prefix.
```

Comment:
please change deviceId to deviceID and apply the same all other occurrences of deviceID